lib/Logger/LogAppenderFile.cpp

```diff
@@ -239,7 +239,8 @@ void LogAppenderFile::reopenAll() {
     std::string backup(filename);
     backup.append(".old");
 
-    FileUtils::remove(backup);
+    // TODO Is it acceptable here to ignore failures?
```

Comment:
If removing itself fails, it is acceptable, because we will be overwriting the target file anyway.
However, we don't check if the following _rename_ works or not. If that fails, we do have a problem. However, we can't really report this here because we are inside the logging already :cry: